### ISSUE TITLE
KW_Radio has no documentation

### ISSUE THREAD

**Issue by mobius3** [mobius3/kiwi#56]
New widget, KW_Radio, has no documentation. @Emersont1, since it was you who added it, maybe it is better if you create it? :)

**Comment by Emersont1** [comment]
Yeah, sure I'm currently trying to work on getting a 2D radio button which is proving trickier than i first anticipated, but as soon as i've got that sorted, i'll PR in the documentation
